README.md

```diff
@@ -37,19 +50,43 @@ runner:
   tags: [<tag 1>, <tag 2>, ..] # replace with your own runner tags
 ```
 
-Find out about all configuration options by taking a look at the JSON schema of the configuration. You can generate an up-to-date JSON schema by running:
-
-```bash
-pdm run python -m scripts.generate_config_schema_json
-```
+See [Generating up-to-date Configuration Schema](#generating-up-to-date-configuration-schema)
 
 Start the runner using
 
 ```bash
 python -m askui_runner -c <path to your config file, e.g., askui-runner.config.yaml>
 ```
 
-Currently, the standard logging output of the AskUI runner is minimal - we are soon going to change that. But you should see the runner starting the running of workflows as soon as you schedule some runs through the AskUI Studio.
+## Start UiController
+If you want to run your workflows on the same system as the runner you need to start an UiController that listens on port `6769`. Please download the one for your operating system and start it: [Windows](https://files.askui.com/releases/askui-ui-controller/latest/win32/x64/askui-ui-controller.exe) | [macOS(intel)](https://files.askui.com/releases/askui-ui-controller/latest/darwin/x64/askui-ui-controller.dmg) | [macOS(silicon)](https://files.askui.com/releases/askui-ui-controller/latest/darwin/arm64/askui-ui-controller.dmg) | [Linux](https://files.askui.com/releases/askui-ui-controller/latest/linux/x64/askui-ui-controller.AppImage)
+
+### Execute Workflows on a Remote System: Change UiController URL
+You can change the UiController-URL so the runner can talk to a UiController that runs on a remote machine or on a different port: 
+
+```yml
+...
+runner:
+  ...
+  controller:
+    host: "127.0.0.1"
+    port: 7000
+```
+
+## Generating up-to-date Configuration Schema
+
+Requirements:
+- [PDM](https://pdm.fming.dev/latest/) 2.8 or higher for contributing and creating the JSON schema of the config
+
+Find out about all configuration options by taking a look at the JSON schema of the configuration. You can generate an up-to-date JSON schema by cloning this repository and running the following commands.
+
+```bash
+## Install and initialize pdm
+pip install pdm
+pdm install
+
+pdm run python -m scripts.generate_config_schema_json
```

Comment:
Would be awesome if we had that directly available to the user through our docs in a nicely readable form.

Reply:
I would like that 👍 

Put it on my list ❤️